docs/external plugins/Verifier/sbom.md

```diff
@@ -0,0 +1,294 @@
+# SBOM Validation
+
+This document outlines how Ratify can be used to verify sbom (Software bill of material). The `sbom` verifier is added as a plugin to the Ratify verification framework.  Currently the sbom verifier is in 2.0.0-alpha release, and it supports the following sbom validation:
+- sbom attached to the subject image as a referrer artifact
+- sbom generated in spdx+json format
```

Comment:
I think we just mention it's in `SPDX` format and link to the SPDX website if there is. In my opinion, I think `spdx+json` is confusing.